src/core/Link/styles.ts

```diff
@@ -0,0 +1,26 @@
+import { fade, makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    color: theme.palette.text.primary,
+    fontWeight: 'normal',
+    textDecorationLine: 'none',
+    '&:hover': {
+      color: theme.palette.secondary.main,
+      boxShadow: `0px 4px 4px -2px ${fade(theme.palette.highlight, 0.7)}`,
```

Comment:
Using `em` in place of `px` would be preferred

Reply:
For this use case we are using the exact same box shadow of MUI, shadow weight 7 so we have copied over the exact style that they use, We can convert the pixel values to its equivalent rem but I think this should work as an exception.

Reply:
@S-ayanide @AVRahul  
I've created the box shadow in the exact style as it's on the button style. Can change the px with **_rem or em_**

Reply:
Yes, I think keeping it `px` is good for box shadows, since its exactly similar to MUI's shades. (An exception only for box-shodow)